### DISCUSSION TITLE
Welcome to Watch app for Tesla Discussions!

### DISCUSSION THREAD

**Discussion by Kemmey** [Kemmey/Kemmey-TeslaWatch-Public#237] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emi777ly** [comment]
What does it mean: "allow wake in background"? 
Is it going to keep car awake constantly and drain the battery?
![image](https://user-images.githubusercontent.com/7103983/121825039-128e2200-cc65-11eb-97ca-833c9682e30f.jpeg)

**Reply by Kemmey** [reply]
I basically allows the app to wake up in the background in order to do some housekeeping. If you have the widgets enabled, it will grab new data from the vehicle to show on those. If you have any of the reminders active, it will do the same and process any potential notifications. 

You can control background updates and wake separately. So you could have the app do background updates somewhat frequently, but not allow it to wake your car - then it would only get new data if your car was already awake, like if sentry mode was on. 

In reality, the defaults drain very little. On my Model 3 long range, it's about 1-2 % a day - and not all of that is attributable to the app. If you had sentry mode on, that alone would drain so much more. 

What I do like to do, though, is set the background updates to 1 hour, but then only allow wake when I'm awake myself - that way I get updated widgets and complications, but the car can go to deep sleep during the night.

**Reply by emi777ly** [reply]
Thank you 👍🏻

**Comment by emi777ly** [comment]
Can I add charging schedule and depart time from your application?

**Reply by Kemmey** [reply]
Unfortunately not. Those are not exposed by Tesla. 

What you could do is to build some custom Shortcuts that does the charging schedule for you - but I highly recommend against that. First, scheduling on iOS is unreliable at best, second, if car could not be woken up, the app can't start charging and you'd wake up to an empty vehicle in the morning. 

If you'd like more control over charging, I highly recommend some other apps that control the vehicle via their own server implementations so you're not reliant on your own phone for the scheduling. 

Charged app, Stats app or Teslascope come to mind

**Reply by emi777ly** [reply]
Thank you

**Comment by mlgpbp01** [comment]
I am using Watch for Apple App and cannot get the SIRI to access shortcut on the Apple Watch series 6 - i did set the shortcut on the Apple watch but it seems to not be syncing (Just the shortcuts) with the Apple watch - i rebooted multiple times both devices but had no luck. The SIRI shortcut works fine on the iPhone using SIRI Voice control - any help will be appreciated (Tesla Model 3)

**Reply by Kemmey** [reply]
Hey,

Hmm, sounds like you've already done all the basic troubleshooting without luck. 

When you open up the shortcuts app on your watch, is it completely empty?

Try opening the shortcuts app on your phone, find one of the custom phrases you configured in the app and tap the three dots button to edit it. Then tap the top corner three dots button to see details and finally verify the show on Apple Watch button is on

![image](https://user-images.githubusercontent.com/4688084/124192237-3e166680-dac5-11eb-9e77-c37ee89d5178.jpeg)

![image](https://user-images.githubusercontent.com/4688084/124192246-42db1a80-dac5-11eb-80ad-2a600b836124.jpeg)

**Comment by mlgpbp01** [comment]
Ok so the short cut is empty on the Apple Watch. Although on the phone the short cuts are working and are set to sync with Apple Watch 

Pinakin Purohit

**Reply by mlgpbp01** [reply]
Yes completely empty not a single short cut thanks

**Reply by Kemmey** [reply]
It seems like you're not alone with this problem. I found this post by searching for "shortcuts not syncing to apple watch":

https://www.reddit.com/r/shortcuts/comments/iumfox/shortcuts_are_not_syncing_between_iphonewatch_on/

There's a comment with what looks like a solution:

I found the solution! It involves deleting and reinstalling the Shortcuts app. Take these steps to prevent losing all your shortcuts:

Go to Settings > Shortcuts > iCloud Sync and turn it on. Give the app a few minutes to sync up to make sure all your shortcuts are uploaded to iCloud.
Your shortcuts will be saved to iCloud, but your automations will NOT. Take screenshots of what automations you have and, especially for long and complicated ones, of the contents of the automations. This will make it much easier to restore your automations.
Delete Shortcuts app and reinstall from App Store. Your shortcuts will be restored from iCloud (again, your automations will not and you will have to make them again).
Turning on iCloud Sync (if it wasn't on before) may have removed some shortcuts from their folders. You can move them back now.
Tapping the "..." button on any shortcut may bring up a menu offering you to fix a sync issue by choosing the version saved to either your phone or your watch. Choose the one on your phone. This is because any changes you made on your phone wouldn't have been saved on the watch, since it wasn't syncing right.
The Shortcuts app on Apple Watch should now be synced! Any changes you make on your phone will now be properly reflected on your watch.

I don't know whether turning of iCloud Sync for the Shortcuts app would stop it from syncing to your watch, but I'm definitely not going to try it.

**Comment by mlgpbp01** [comment]
Thanks will give it a short. Appreciate your prompt reply 

Pinakin Purohit

**Comment by mlgpbp01** [comment]
Hi Kemmey, thanks for the suggestion. Deleting and re-adding Shortcut did the trick. All is working on my Apple watch now. I can RUN the commands for Tesla using SIRI on my Apple Watch. Appreciate your prompt attention.

**Comment by amy26point2** [comment]
Loving the app’s functionality. Would it be at all possible to enable dog mode from Siri or the watch? Thanks!

**Reply by Kemmey** [reply]
I would love to - but Tesla currently does not support this unfortunately. Once they add it, I'll add support as well.

**Reply by amy26point2** [reply]
Appreciate the response. Thank you!

**Comment by amy26point2** [comment]
Would it be possible to have a Siri shortcut to turn on and off seat heaters, or am I maybe overlooking it?

**Comment by Kemmey** [comment]
@amy26point2

Possibly, yes - currently implemented, no. 

I'll add it to the list for next version! :-D

**Comment by EV4Me** [comment]
Hello,
This is an easy one, searched but could not find the answer. 
How to turn off all tones on watch app?
Sorry if this has been asked previously. Thank you!

**Reply by Kemmey** [reply]
Unfortunately this one's not so easy. Apple literally will not allow a haptic-only feedback on Apple Watch. So it always makes a sound as well, unless you change it for the entire watch.

**Comment by goatea** [comment]
Hi there.  Just got the app. I turned off my phone to see if it would work just with the watch (the reason I purchased the app). It wanted the key card. Will I need the key card each time? Or just this first time? Thanks.

**Reply by Kemmey** [reply]
Hey,

You'll need to tap the unlock button on the watch to unlock. Walk-away lock and unlock only works with Bluetooth on your phone with the official app. 

But you can do all the same things with the watch app - you'll just have to push the buttons manually.

**Comment by wombat996** [comment]
Any idea if the app would work utilizing a watch set up via family setup? The iPhone used to host the family setup would have the app installed, and the person using the watch would have the app as well, just not their own personal iPhone. Thanks! Have loved the app for a few years, but I also like Pixels.

**Reply by Kemmey** [reply]
@Wombat996 I honestly do not know, but I would absolutely expect so. The app is fully independent and can run only on the watch without having ever seen the phone counterpart. Only issue is logging in - you'll have to figure out how to get the token transferred to the watch somehow - if the phone setting up the watch can do that, then problem solved. If not - you should be able to login to same Apple ID as paired to watch on any other device, and token should transfer with iCloud Keychain.

**Comment by terenceho** [comment]
I missed the discount period last week.😭 May I know the next discount period? Thanks mate!

**Comment by heather910906** [comment]
When is the update for opening the door?
